src/script/view_model/ContentViewModel.ts

```diff
@@ -401,17 +466,17 @@ export class ContentViewModel {
         this.conversationRepository.active_conversation(null);
       }
 
-      return this.messageList.release_conversation();
+      return this.messageList.release_conversation(undefined);
```

Comment:
Why did you add `undefined` here?

Reply:
messageList.release_conversation excepts an argument but it's handling it if nothing is passed so typescript compiler was unhappy about this.